doc/sphinx/localization.rst

```diff
@@ -21,6 +21,23 @@ This lets us translate strings in various places:
 
   The ``trimmed`` argument is important as it ensures leading and trailing whitespace and newlines are not included in the translation strings (they mess up the translation system).
 
+Localized Models
+~~~~~~~~~~~~~~~~
+
+Currently the following models use translations:
+
+* License (field ``name``) in ``deposit.models``
+
+For localization we use `django-vinaigrette <https://pypi.org/project/django-vinaigrette/>`_. Please read their documentation for further information. In short: You have to keep in mind that:
+
+* in admin interface you do not see the localized strings,
+* you should add only english in the admin interface,
+* you will have to recreate the ``*.po`` files and add the translation manually (see below),
+* your local translations do not interact with TranslateWiki.
+
```

Comment:
Given that generating the ``.po`` files is currently handled by the developer (it is not done by Translatewiki), I guess for the production instance we should be able to generate the ``.po`` files that include the strings for the licenses of the protocols we support, right? (That does not mean we need to put that in the docs, just checking I understood the system).

Reply:
For the protocol, as we discussed in chat: Yes.